dspace-rest/src/main/webapp/WEB-INF/web.xml

```diff
@@ -76,6 +76,14 @@
         <param-value>${dspace.dir}</param-value>
     </context-param>
 
+    <context-param>
+        <param-name>log4jConfiguration</param-name>
+        <param-value>${dspace.dir}/config/log4j2.xml</param-value>
+        <description>
+	    The location of the Log4J configuration
+        </description>
```

Comment:
I *think* Servlet API 3.1 requires the `<description>` to be *first* in `<context-param>` (IIRC I had to reorder some old context-params when updating to 3.1).  You may want to double check all the updated `web.xml` files to be certain they are "valid".

Reply:
Correct.